config/default/core.entity_form_display.taxonomy_term.h_notification_type.default.yml

```diff
@@ -27,6 +29,13 @@ content:
     third_party_settings: {  }
     type: options_select
     region: content
+  field_h_icon:
+    type: media_library_widget
+    weight: 6
```

Comment:
Given the importance of the icon field I would suggest we place it after the Colour field in the Notification Type edit form.